target/linux/econet/Makefile

```diff
@@ -19,4 +19,6 @@ endef
 # include the profiles
 include $(INCLUDE_DIR)/target.mk
 
+DEFAULT_PACKAGES += kmod-usb3 kmod-usb-xhci-mtk nand-utils kmod-usb-storage kmod-fs-vfat kmod-fs-msdos
```

Comment:
Please stay consistent with other hardware targets and do not add `kmod-usb-storage` or filesystem drivers to the default package set. Also `nand-utils` seems unrelated and should not be added.
Even `kmod-usb3` and `kmod-usb-xhci-mtk` are better to be added on board-level as there are likely also boards which do not expose any USB port.

Reply:
Ok thank you for the advice. I really don't know what's normal here.

The reason I added these is because SD-Card readers are common on these devices. Does it make sense to add it for the board that definitely does have an SD-Card reader, or should I leave it out anyway?